### PR TITLE
Create add poi

### PR DESCRIPTION
**Pull request by monkeyysups**

### CODE REVIEW

**Comment by monkeyysups** [comment]
closes #43

File: tests/api/test_energy_robotics_api.py [hunk 1]
```diff
@@ -40,6 +48,59 @@ def test_fails_if_status_is_anything_else(self):
             api.pause_current_mission("test_exr_robot_id")
 
 
+@mock.patch(
+    "isar_exr.api.graphql_client.get_access_token",
+    mock.Mock(return_value="test_token"),
+)
+class Test_create_point_of_intrest:
```
Comment (Afonso-2403) [inline tests/api/test_energy_robotics_api.py:55]: Following the other test classes the class name should be in UpperCamelCase: `TestCreatePointOfInterest`

File: tests/api/test_energy_robotics_api.py [hunk 2]
```diff
@@ -40,6 +48,59 @@ def test_fails_if_status_is_anything_else(self):
...
+    expected_return_id = "point_of_interest_type_id"
+    api_point_of_intrest_response: Dict[str, Any] = {
```
Comment (Afonso-2403) [inline tests/api/test_energy_robotics_api.py:57]: There is a typo, it should be `api_point_of_interest_response`

File: tests/api/test_energy_robotics_api.py [hunk 3]
```diff
@@ -40,6 +48,59 @@ def test_fails_if_status_is_anything_else(self):
...
+        )
+        poi: AddPointOfInterestInput = AddPointOfInterestInput(
+            name="mock_name",
+            site="mock_site",
+            frame="mock_frame",
+            type=PointOfInterestTypeEnum.GENERIC,
+            pose=pose,
+            photoAction=action,
+        )
+        with pytest.raises(expected_exception=Exception):
```
Comment (Afonso-2403) [inline tests/api/test_energy_robotics_api.py:100]: Expected exception should be `RobotException`, right?

**Review by Afonso-2403 — CHANGES_REQUESTED** [review]
looks good to me apart from a couple of typos :)

File: tests/api/test_energy_robotics_api.py (hunk 3 above)
Comment (monkeyysups) [reply tests/api/test_energy_robotics_api.py]: could be both, but RobotExcepion is more precise, so fixed it.

File: src/isar_exr/api/energy_robotics_api.py [hunk 4]
```diff
@@ -73,6 +81,98 @@ def pause_current_mission(self, exr_robot_id: str) -> None:
         if not success:
             raise RobotException(f"Invalid status after pausing mission: '{status}'")
 
+    def create_point_of_intereste(
```
[the remark this answers is not in the document]
Comment (monkeyysups) [reply src/isar_exr/api/energy_robotics_api.py]: done

**Review by Christdej — APPROVED** [review]
LGTM

**Review by Afonso-2403 — APPROVED** [review]
lgtm

**Pull request merged**